FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2013, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+@interface FBSnapshotTestCaseDeviceAgnosticTests : FBSnapshotTestCase
+
+@end
+
+@implementation FBSnapshotTestCaseDeviceAgnosticTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.deviceAgnostic = YES;
+    self.recordMode = NO;
+}
+
+- (void)testViewSnapshot
+{
+    UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+    redView.backgroundColor = [UIColor redColor];
+    FBSnapshotVerifyView(redView, nil);
```

Comment:
@nscoding were you checking the demo project or the framework?  The demo project has a test host, which corroborates what I saw when debugging:

![image](https://cloud.githubusercontent.com/assets/444217/11851895/21310916-a403-11e5-889c-dc4404c4520c.png)

As the device agnostic tests I added are in the demo project, I don't think changing the framework test target to have a host will make a difference here.